benchmark/schema/spmv.reuse.json

```diff
@@ -0,0 +1,58 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "spmv.reuse.json",
+  "title": "SpMV",
+  "description": "Reusable SpMV definition",
+  "type": "object",
+  "properties": {
+    "format": {
+      "enum": [
+        "coo",
```

Comment:
Do we also remove the format out of CLI?
some of them might be hard to understand without comment

Reply:
It will be removed from the CLI, but there will be extra documentation to make up for it.